runtime/lua/vim/uri.lua

```diff
@@ -117,19 +117,16 @@ local function uri_to_fname(uri)
   return uri
 end
 
---- Return or create a buffer for a uri.
----@param uri (string): The URI
----@return bufnr.
----@note Creates buffer but does not load it
+--- Get the buffer for a uri.
+--- Creates a new unloaded buffer if no buffer for the uri already exists.
+--
+---@param uri string: The URI
```

Comment:
Elsewhere we've just been using `type` without the colon. I don't have an opinion either way but we should decide on one or other and be consistent.

Reply:
I also don't care, I've been using type without the colon too but see an argument for the colon being more readable.

Reply:
I removed them and in some cases also removed the text, as it felt a bit repetitive.